parsl/monitoring/monitoring.py

```diff
@@ -143,16 +143,16 @@ def __init__(self,
         Parameters
         ----------
         hub_address : str
-             The ip address at which the workers will be able to reach the Hub.
+             The ip address at which the workers will be able to reach the Router.
```

Comment:
This is confusing. hub_address is the address for the Router? While this is correct, it will be confusing for the user who doesn't know that the hub contains a router.